test/IRGen/autorelease.sil

```diff
@@ -1,4 +1,4 @@
-// RUN: %target-swift-frontend(mock-sdk: %clang-importer-sdk) %s -emit-ir | %FileCheck -check-prefix CHECK -check-prefix CHECK-%target-ptrsize -check-prefix %target-cpu -DINT=i%target-ptrsize %s
+// RUN: %target-swift-frontend(mock-sdk: %clang-importer-sdk) %s -enable-objc-interop -emit-ir | %FileCheck -check-prefix CHECK -check-prefix CHECK-%target-ptrsize -check-prefix %target-cpu -DINT=i%target-ptrsize %s
 
```

Comment:
Enabling `-enable-objc-interop` with Wasm target makes internal clang instance confusing, so we need to mark those tests with `objc_codegen` to exclude them from wasm tests
```suggestion
// REQUIRE: objc_codegen

```

Reply:
Oh, that's awkward. I'll have to split the new test into two files in that case.

Reply:
Sorry for my typo: `REQUIRE` -> `REQUIRES` 🤦‍♂️

Reply:
No worries. 🙂